hr_employee_relative/migrations/12.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# Copyright 2019 Tecnativa - Pedro M. Baeza
```

Comment:
🤣🤣🤣

Reply:
Copy-pasted and didnt realize haha, I will fix it now